src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Pages/ResetPassword.razor.cs

```diff
@@ -76,18 +76,12 @@ private bool ValidateResetPassword()
 
     private async Task Submit()
     {
-        if (IsLoading)
+        var isResetPasswordInvalid  = ValidateResetPassword() is false;
```

Comment:
why did you define a new variable here?
I think it's redundant. the `ValidateResetPassword() is false` is more expressive.